roles/beaker/data/distros/tasks/main.yml

```diff
@@ -3,19 +3,22 @@
 - name: import user defined distros
   command: beaker-import {{ item }}
   with_items: "{{ user_distros }}"
+  ignore_errors: yes
 
 - name: import centos
   command: beaker-import {{ item }}
   with_items:
     - http://mirror.centos.org/centos/6/os/x86_64/
     - http://mirror.centos.org/centos/6/os/i386/
     - http://mirror.centos.org/centos/7/os/x86_64/
+  ignore_errors: yes
 
 - name: import Fedora releases
   command: beaker-import --family=Fedora --arch=x86_64 {{ item }}
   with_items:
-    - http://download.fedoraproject.org/pub/fedora/linux/releases/29
+    - http://download.fedoraproject.org/pub/fedora/linux/releases/31
     - http://download.fedoraproject.org/pub/fedora/linux/releases/30
```

Comment:
Can you sort it? please :crying_cat_face:

Reply:
Yep, I'll change it to SORT ASC